my-world/app/routes/mcu/index.tsx

```
@@ -0,0 +1,181 @@
+import { HeroBanner } from '~/components/index_page/hero_banner';
+import WatchOrder from '~/components/watch_order/watchOrder';
+import { loadProgress } from '~/utils/db';
+
+export function meta({}: Route.MetaArgs ) {
+  return [
+    { title: "A Monument to the Marvel Cinematic Universe"},
+    { name: "description", content: "Rewatch cinematic glory and participate in a fun ranking of the Marvel Cinematic Universe"},
+  ]
+}
+
+export async function loader({} : Route.LoaderArgs){
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)